frontend/components/Layouts/BlogPageLayout.tsx

```diff
@@ -0,0 +1,88 @@
+import { format } from 'date-fns'
+import { getUTCDate } from '../../utils'
+
+type Props = {
+  date: string
+  title: string
+  image: string
+  children: React.ReactNode
+}
+
+const BlogPageLayout = (props: Props) => (
+  <div>
```

Comment:
I'd also like to stick to being nice and explicit with our React components typings, so like I've been doing in all the recent PRs: `const BlogPageLayout: React.FC<Props> = () => {...}`

Also, and again a lot of this is just a matter of style and convention/continuity across the app, what do you think about destructuring the props in a case like this? 

`const BlogPageLayout: React.FC<Props> = ({ title, image, date, children }: Props) => {...}` ?

Reply:
That all sounds good to me! If we have `React.FC<Props>`, then we can omit the second `: Props`

Reply:
Yep I meant to edit that out this morning haha